internal/sdkprovider/service/clickhouse/clickhouse_grant.go

```diff
@@ -107,6 +107,9 @@ func ResourceClickhouseGrant() *schema.Resource {
 		DeleteContext: resourceClickhouseGrantDelete,
 		Schema:        aivenClickhouseGrantSchema,
 		Timeouts:      schemautil.DefaultResourceTimeouts(),
+		Importer: &schema.ResourceImporter{
```

Comment:
Looks like it has some conflicts with [another](https://github.com/aiven/terraform-provider-aiven/pull/2138/files#diff-cbe84f9ccb8893bfe143f754cae1383fc3c26af3e14eac485e6453a7437d7cb7R110) PR which is related to the grant.